### PR TITLE
Full screen navigation

### PR DESCRIPTION
**Pull request by RaulMorales404**
# Description
 
 
 - What?: Implemented navigation of all app views 
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: So that users can move around in the application.
 
  ## Screenshots 
navigation

### CODE REVIEW

File: GoToProject/App.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+        <Stack.Screen name="Login"  component={Login} options={{header: () => null}} />
+        <Stack.Screen name="SignUp"  component={SignUp}  options={{header: () => null}} />
+        <Stack.Screen name="SelectLocation"  component={SelectLocation}  options={{header: () => null}} />
+        <Stack.Screen name="Home"  component={Home}  options={{header: () => null}} />
+        <Stack.Screen name="DetailsScreen"  component={DetailsScreen }  options={{header: () => null}} />
+        <Stack.Screen name="Reactions"  component={Reactions}  options={{header: () => null}} />
+        <Stack.Screen name="AddNewSite"  component={AddNewSite}  options={{header: () => null}} />
+        <Stack.Screen name="WorkShedules"  component={WorkShedules}  options={{header: () => null}} />
+        <Stack.Screen name="SiteInformation"  component={SiteInformation}  options={{header: () => null}} />
+        <Stack.Screen name="SelectImagesSite"  component={SelectImagesSite}  options={{header: () => null}} />
```
Comment (YilKanda) [inline GoToProject/App.js:29]: Can you reduce the options of the header?

File: GoToProject/src/components/home/categoris/index.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const Categories = () => {
           {categories.map(value => {
             return (
               <ButtonCategori key={value.id}>
-                <TextButton fontt="100" clt="#8B8FA4">
+                <TextButton font="17px" fontt="100" clt="#8B8FA4">
```
Comment (YilKanda) [inline GoToProject/src/components/home/categoris/index.js:23]: Can you implement a color library?

File: GoToProject/App.js (hunk 1 above)
Comment (RaulMorales404) [reply GoToProject/App.js]: yes thank you very much for your comment

File: GoToProject/src/components/home/categoris/index.js (hunk 2 above)
Comment (RaulMorales404) [reply GoToProject/src/components/home/categoris/index.js]: done

**Review by 96vladi — APPROVED** [review]
good job

**Review by EusebioAjas — APPROVED** [review]

File: GoToProject/App.js [hunk 3]
```diff
@@ -14,19 +14,20 @@ import {createNativeStackNavigator} from '@react-navigation/native-stack';
...
+        <Stack.Screen name="Login" component={Login} options={header} />
+        <Stack.Screen name="SignUp" component={SignUp} options={header} />
+        <Stack.Screen name="SelectLocation" component={SelectLocation} options={header} />
+        <Stack.Screen name="Home" component={Home} options={header}/>
+        <Stack.Screen name="DetailsScreen" component={DetailsScreen} options={header} />
+        <Stack.Screen name="Reactions" component={Reactions} options={header} />
+        <Stack.Screen name="AddNewSite" component={AddNewSite} options={header} />
+        <Stack.Screen name="WorkShedules" component={WorkShedules} options={header} />
+        <Stack.Screen name="SiteInformation" component={SiteInformation} options={header} />
+        <Stack.Screen name="SelectImagesSite" component={SelectImagesSite} options={header} />
```
[the remark this answers is not in the document]
Comment (RaulMorales404) [reply GoToProject/App.js:30]: the message has just been added that I will implement it in another branch.

**Pull request merged**